### ISSUE TITLE
User Curation Page - Make tool to replace an account's username

### ISSUE THREAD

**Issue by tiwane** [inaturalist/inaturalist#3966]
In the course of iNaturalist events, we come across accounts with usernames that are hate speech or sexually explicit and are often suspended by curators. Because that username appears on content they posted that may not be hidden, I like to change the username of the account but there's no UI to do that. 

On the User Curation page (`https://www.inaturalist.org/admin/users/[username]`) can we 

- [ ]  add a button that says "Replace Username"

- [ ] when that button is clicked, display a modal that asks if we want to replace the account's username with an anonymized on. Its two buttons should say "Yes, Replace" and "Cancel".

- [ ] if one presses "Yes, Replace", the username will be changed to `naturalistXXXX` where the X's are numbers and the username is unique.

**Comment by jotamayo97** [comment]
I see that only admins (and not curators) can access` https://www.inaturalist.org/admin/users/[username]`. Am I misunderstanding something? Maybe there should also be a button in the Admin Tools on the /people page? As far as I know, curators can access that one

![Image](https://github.com/user-attachments/assets/50c7d923-e661-4cc0-bf0f-7f626a680540)

**Comment by kueda** [comment]
Correct, only admins can access that page. This issue is for making a tool accessible from that page that allows admins to change a user's login.
